libraries/AP_NavEKF/EKFGSF_yaw.cpp

```diff
@@ -630,6 +630,11 @@ bool EKFGSF_yaw::getYawData(ftype &yaw, ftype &yawVariance) const
     if (!vel_fuse_running) {
         return false;
     }
+    // at least three models must be in good health for a result to be
+    // valid:
+    if (N_MODELS_EKFGSF - n_clips < 3) {
```

Comment:
i think this should be a parameter, or a return reference param

Reply:
> i think this should be a parameter, or a return reference param

I've created #20726 with that alternative; went with a pointer so the EKF didn't need to be modified.